### PR TITLE
Updating Screenshot to reflect latest version.

### PR DESCRIPTION
**Pull request by mark-robustelli**
Updating screenshot to reflect latest version of FusionAuth.

### CODE REVIEW

**Comment by mooreds** [comment]
What issue is this for, @mark-robustelli ?

also, who is @DevUndefined ? Is that your personal account?

File: site/assets/img/docs/email-templates/email-settings.png
Comment (mooreds) [inline site/assets/img/docs/email-templates/email-settings.png:1]: Please make sure this is shrunk using tinypng.

**Review by mooreds — CHANGES_REQUESTED** [review]
While you are in there, can you please add the missing fields for the SMTP settings?

We are missing the `default from address` and the `debug enabled`. I typically use the value of the admin tooltip as a starting point, for this documentation.

File: site/assets/img/docs/email-templates/email-settings.png
Comment (mark-robustelli) [reply site/assets/img/docs/email-templates/email-settings.png:1]: Do you see something that leads you to believe it is not or is this a double check?  I used the script so it should be, but if not, then I will have to see what is going on there.

Comment (mooreds) [reply site/assets/img/docs/email-templates/email-settings.png:1]: This is just me double checking. If you used the script, you should be fine.

**Comment by mark-robustelli** [comment]
> also, who is @DevUndefined ? Is that your personal account?

Okay got this figured out...when I got the warning from github about submitting a commit with my private email address...I had to change to use the github no reply address.  With new usernames you have to add the id. (i.e. 137117976+mark-robustelli@users.noreply.github.com)  I got it but goofed when pasting to the git config.   my id is 137117976.  I used 37117976.  Sure enough, if I look up @DevUndefined that is their user ID.  Weird that it would let you post that.  I will see if I can figure out how to back that out or maybe just close this request and start a fresh one.

**Comment by mooreds** [comment]
Sounds good!

**Comment by mark-robustelli** [comment]
closing this due to @DevUndefined being listed.  This was my mistake when adding the no reply user.

**Pull request closed without merging**